StRoot/StFcsDbMaker/StFcsDb.h

```diff
@@ -174,7 +174,8 @@ class StFcsDb : public TDataSet {
   //! get the STAR frame cooridnates from other way
   StThreeVectorD getStarXYZfromColumnRow(int det,float col, float row, float FcsZ=-1.0) const; //from column/row[cell size unit]
   StThreeVectorD getStarXYZ(int det, int col, int row, float FcsZ=-1.0) const;   //from column/row [cell unit]
-  StThreeVectorD getStarXYZ(StFcsHit* hit, float FcsZ=-1.0) const;                   //from StFcsHit
+  StThreeVectorD getStarXYZ(StFcsHit* hit, float FcsZ=-1.0) const;               //from StFcsHit
+  StThreeVectorD getStarXYZ(StFcsCluster* clu, float FcsZ=-1.0) const;           //from StFcsCluster
```

Comment:
I'd define the parameter as const

```suggestion
  StThreeVectorD getStarXYZ(const StFcsCluster* clu, float FcsZ=-1.0) const;           //from StFcsCluster
```

Reply:
Done. For both with StFcsClutser and StFcsHit.